docs/APIExamples.md

```diff
@@ -0,0 +1,70 @@
+# API Code snippets
+
+## Converting between MLModel and Spec
+ 
+```python
+import coremltools
+
+# Load MLModel
+mlmodel = coremltools.models.MLModel('path/to/the/model.mlmodel')
+
+# use model for prediction
+mlmodel.predict(...)
+
+# save the model
+mlmodel.save('path/to/the/saved/model.mlmodel')
+
+# Get spec from the model
+spec = mlmodel.get_spec()
+
+# print input/output description for the model
+print(spec.description)
+
+# save out the model directly from the spec
+coremltools.models.utils.save_spec(spec,'path/to/the/saved/model.mlmodel')
+
+# convert spec to MLModel, this step compiles the model as well
+mlmodel = coremltools.models.MLModel(spec)
+
+# Load the spec from the saved .mlmodel file directly
+spec = coremltools.models.utils.load_spec('path/to/the/model.mlmodel')
+```
+
+## Visualizing the CoreML model
+```python
+import coremltools
+
+mlmodel = coremltools.models.MLModel('path/to/the/model.mlmodel')
```

Comment:
I would also call this variable `neural_network` rather than just `mlmodel`.

Reply:
Possibly, although the model that is returned by MLModel is the entire mlmodel object which contains the neural network spec as one of its components (in this case though that is the only component)

Reply:
I have made it "nn_mlmodel" instead.